go.mod

```diff
@@ -114,3 +114,5 @@ require (
 	google.golang.org/protobuf v1.34.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/openshift-online/ocm-sdk-go => github.com/venkateshsredhat/ocm-sdk-go v0.1.445-newtest
```

Comment:
Not required , use the latest https://github.com/openshift-online/ocm-sdk-go/releases/tag/v0.1.447